python/runtime/cudaq/platform/py_alt_launch_kernel.cpp

```diff
@@ -493,6 +493,18 @@ py::object pyAltLaunchKernelR(const std::string &name, MlirModule module,
   return returnValue;
 }
 
+/// @brief Helper function to get boolean environment variable
+static bool getEnvBool(const char *envName, bool defaultVal = false) {
```

Comment:
Is this function copied from somewhere else? It looks familiar.

If it is a copy, we should just put it someplace "common" and expose the function to all the libraries that use it.

Reply:
will do, thanks

Reply:
done